addons/survey/wizard/survey_invite.py

```diff
@@ -212,13 +212,13 @@ def _send_mail(self, answer):
         else:
             mail_values['email_to'] = answer.email
 
-        # optional support of notif_layout in context
-        notif_layout = self.env.context.get('notif_layout', self.env.context.get('custom_layout'))
-        if notif_layout:
+        # optional support of email_layout_xmlid in context
+        email_layout_xmlid = self.env.context.get('email_layout_xmlid', self.env.context.get('notif_layout'))
```

Comment:
Too bad you removed the other one, it would have looked very nice with 3 context values following each other (.. or not :D )